Assets/GothicVR/Scripts/Creator/MeshCreator.cs

```diff
@@ -471,5 +471,16 @@ private static void CreateBonesData(GameObject root, SkinnedMeshRenderer rendere
             renderer.sharedMesh.bindposes = bindPoses;
             renderer.bones = bones;
         }
+        
+        private Material GetEmptyMaterial()
+        {
+            var standardShader = Shader.Find(defaultShader);
```

Comment:
Check if defaultShader is loaded correctly as right now the shader cannot be found on builds and returns null

Reply:
Found a way to reference it: https://stackoverflow.com/questions/36909567/textures-loads-in-editor-but-not-in-standalone-appears-pink